src/site/content/en/blog/web-otp/index.md

```diff
@@ -150,7 +150,7 @@ The whole process is diagrammed in the image below.
   <figcaption class="w-figcaption w-figcaption--fullbleed">
```

Comment:
We should replace the diagram image as well. I'll work on it.

Reply:
Thank you.

Reply:
The image is replaced.